pkg/game/board.go

```diff
@@ -439,11 +441,47 @@ func (board *board) ScoreRoads(placedTile elements.PlacedTile) elements.ScoreRep
 	for _, road := range roads {
 		// check if the side of the tile was not already checked (special test case reference: TestBoardScoreRoadLoopCrossroad)
 		if !checkedRoadSides.OverlapsSide(road.Sides) {
-			scoreReportTemp, roadSide := board.ScoreRoadCompletion(placedTile, road)
+			scoreReportTemp, roadSide := board.ScoreRoadCompletion(placedTile, road, forceScore)
 			scoreReport.Join(scoreReportTemp)
 			checkedRoadSides |= roadSide
 		}
 		checkedRoadSides |= road.Sides
 	}
 	return scoreReport
 }
+
+func (board *board) ScoreFinalMeeples() elements.ScoreReport {
+	meeplesReport := elements.NewScoreReport()
+
+	// score cities first (because they have their own manager)
+	meeplesReport.Join(board.cityManager.ScoreCities(true))
```

Comment:
but technically, do they need to me removed? Manager knows each city, and is able to score each once, so there is no risk of counting in multiple times.

Reply:
Technically no but I think it would be good to keep this consistent and avoid any unwanted surprises, in case it were to matter in some way in the future.

Reply:
done